src/Modules/OrchardCore.Commerce/Assets/Scripts/discount.js

```diff
@@ -0,0 +1,7 @@
+const prices = Array.from(document.querySelectorAll('.tax-part-gross-price-value, .price-part-price-field-value'));
+
+function strikeOutPrice(price) {
+    price.innerHTML = "<del class=\"text-danger\">" + price.textContent + "</del>";
```

Comment:
According to ESLint:

    4:23  warning  Strings must use singlequote                         quotes
    4:75  warning  Strings must use singlequote                         quotes